resources/examples/simple_cluster/playbooks/login1.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: login1
-  hosts: "{{ target | default('login1') }}"
+  hosts: "login1"
```

Comment:
Why single host target instead of login group ?
Documentation must also be changed according to this.

Reply:
Because I find it dangerous to use target. When adding "target", I didn't knew --limit argument.

I prefer to force a static host or group of hosts in the playbook. Then use --limit to reduce scope.

In production, having a flexible playbook prove to be dangerous, I have seen that just 3 days ago on a cluster I am working on :-)

Yes, Documentation need to be updated, I will do that in a next commit.

Reply:
Maybe use a group (`mg_logins`) instead of a host (`login1`) in the example?

Reply:
@actatux yes, that was the point of my suggestion, use login group in hosts field and restrict to login1 using limit option,  i guess this playbook isnt dedicated only to the first login and can be applied on all login nodes, right ?

Reply:
Agree. We might need to rename it `logins.yml`.

Reply:
Oh, sorry @bouriquet 

I miss read your comment. Thought you were referring to the removal of target.

Then we are all in line :-)